ethereum/evmtool/build.gradle

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+apply plugin: 'java-library'
+apply plugin: 'application'
+apply plugin: 'idea'
+
+jar {
+  baseName 'pantheon-evmtool'
```

Comment:
Maybe we need to change pantheon by Besu here

Reply:
Yep.  The code for this was sitting in my old pushes that long ago.  Fixed.